components/bitbucket/new-pipeline-event.js

```diff
@@ -0,0 +1,83 @@
+const _ = require("lodash");
```

Comment:
lodash is [kinda big](https://bundlephobia.com/result?p=lodash@4.17.20) -- if you are just using `_.isEmpty` maybe use [lodash.isempty](https://bundlephobia.com/result?p=lodash.isempty@4.4.0)? This will speed up async option configuring + deploys so it's a good pattern to try to adhere to whenever it's easy.

Reply:
I will do some refactoring. I avoided using per-method packages since it's [discouraged by Lodash](https://lodash.com/per-method-packages), but I can definitely do something to avoid requiring the whole package.